### PR TITLE
Add 0.24.0 to CHANGELOG

### PR DESCRIPTION
**Pull request by sdogruyol**
Adds 0.24.0 release notes to `CHANGELOG`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+- **(breaking-change)** Flate::Reader.new(&block) and Flate::Writer.new(&block) now use the name open ([#4887](https://github.com/crystal-lang/crystal/pull4887/))
+- **(breaking-change)** Use an Enum for Process stdio redirections ([#4445](https://github.com/crystal-lang/crystal/pull/4445))
+- **(breaking-change)** Remove '$0' special syntax
+- **(breaking-change)** Remove bare array creation from multi assign (a = 1, 2, 3) ([#4824](https://github.com/crystal-lang/crystal/pull/4824))
+- **(breaking-change)** Rename skip macro method to skip_file ([#4709](https://github.com/crystal-lang/crystal/pull/4709))
+- **(breaking-change)** StaticArray#map and Slice#map now return their same type instead of Array ([#5124](https://github.com/crystal-lang/crystal/pull/5124))
+- **(breaking-change)** Tuple#map_with_index now returns a Tuple. ([#5086](https://github.com/crystal-lang/crystal/pull/5086))
+- Packages built with LLVM 3.9.1. They should (hopefully) fix [#4719](https://github.com/crystal-lang/crystal/issues/4719)
+- Syntax: Allow flat rescue/ensure/else block in do/end block ([#5114](https://github.com/crystal-lang/crystal/pull/5086/5114))
```
Comment (hinrik) [inline CHANGELOG.md]: The URL here should be https://github.com/crystal-lang/crystal/pull/5114.

Comment (sdogruyol) [reply CHANGELOG.md]: @Haar thank you :+1:

**Comment by larubujo** [comment]
maybe correct way was commit this in 0.24.0 branch and merge to master...

**Comment by Sija** [comment]
@larubujo nope, `0.24.0` branch was forked **from** the `master`, not the other way around.

**Pull request merged**